tmk_core/common/chibios/timer.h

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2021 Simon Arlott
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include <stdint.h>
+
+// The platform is 32-bit, so prefer 32-bit timers to avoid overflow
+#define FAST_TIMER_T_SIZE 32
+
+#include_next "timer.h"
```

Comment:
This would be implemented the wrong way round, the common timer.h should include all the other implementations per the recent wait/gpio/print work.

Reply:
I've done this now but it has to be `_timer.h` otherwise it takes precedence for `timer.h` and the #include order is going to vary between the two `timer.h` files depending on which one is first.

Reply:
Thats fine, right now its the convention ive used right now for "internal implementation" in lieu of anything else while getting the RiotOS port going.